charts/kubeclarity/templates/service.yaml

```diff
@@ -19,5 +19,11 @@ spec:
       port: {{ index .Values "kubeclarity-runtime-scan" "resultServicePort" }}
       protocol: TCP
       targetPort: {{ index .Values "kubeclarity-runtime-scan" "resultServicePort" }}
+    {{- if ((.Values.kubeclarity).prometheus).enabled }}
```

Comment:
Please remove the unnecessary brackets
`{{- if .Values.kubeclarity.prometheus.enabled }}`

Reply:
The brackets make it nil safe. So i don't have to check 
```
{{ if .Values.kubeclarity }}
  {{ if .Values.kubeclarity.prometheus }}
    {{ if .Values.kubeclarity.prometheus.enabled }} 
...
```

Reply:
I see, we don't do it in any other places, I don't want to start doing it here. we assume the path exists.

Reply:
i will change according to this